momento-sdk/src/main/java/momento/sdk/messages/ListSigningKeysResponse.java

```diff
@@ -1,34 +1,34 @@
 package momento.sdk.messages;
 
 import java.util.List;
-import java.util.Optional;
-import momento.sdk.SimpleCacheClient;
+import momento.sdk.exceptions.SdkException;
 
-/** Response object for list of signing keys. */
-public final class ListSigningKeysResponse {
+/** Response for a list signing keys operation. */
+public interface ListSigningKeysResponse {
 
-  private final List<SigningKey> signingKeys;
-  private final Optional<String> nextPageToken;
+  /** A successful list signing keys operation. Contains the discovered signing keys. */
+  class Success implements ListSigningKeysResponse {
```

Comment:
this reminds me - keep an eye out for if there is any actual signing code in this repo.  The control plane operations for signing keys are fine, but if there are methods to actually do signing, we want to remove those for now and eventually put them in a separate package.  I took a quick scan and didn't see them but just something to keep an eye out for.